sdk/translation/Azure.AI.Translator.Document/src/Custom/MultipartFormFileData.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.AI.Translator.Document
+{
+    /// <summary>
+    /// Represent a multipart form file data.
+    /// </summary>
+    public class MultipartFormFileData
+    {
+        /// <summary>
+        /// Construct a form file data.
+        /// </summary>
+        /// <param name="name">Name of the file</param>
+        /// <param name="content">Content of the file.</param>
+        /// <param name="contentType">Content type of the file.</param>
+        public MultipartFormFileData(string name, BinaryData content, string contentType)
+        {
+            Argument.AssertNotNull(name, nameof(name));
```

Comment:
Validate with the service team if this is optional or not.  I heard it can be optional?

Reply:
I believe it is okay be to compulsory.